discord/colour.py

```diff
@@ -226,5 +226,13 @@ def blurple(cls):
     def greyple(cls):
         """A factory method that returns a :class:`Colour` with a value of ``0x99aab5``."""
         return cls(0x99aab5)
+    
+    @classmethod
+    def darker_greyple(cls):
+        """A factory method that returns a :class:`Colour` with a value of ``0x36393F``.
+        Which is the value of discords dark-theme background."""
```

Comment:
- Documentation should avoid using poorly formed run-on sentences.
- Reference to Discord as a company should be capitalized, and an apostrophe is required here (`Discord's`).
- Explicit reference should be made to the Discord web/desktop client, as the background color varies between the web and mobile native clients.
- "Dark theme" is not one word and thus doesn't need to be hyphenated.